processor/src/operations/io_ops.rs

```diff
@@ -195,16 +195,20 @@ impl Process {
 
         // get the address from position 12 on the stack
         let ctx = self.system.ctx();
-        let clk = self.system.clk();
+        let _ = self.system.clk();
```

Comment:
Let's restore this line and use `clk` instead of `self.chiplets.clk` (see next comment)